lib/handleRequest.js

```diff
@@ -103,7 +103,7 @@ function preValidationCallback (err, request, reply) {
 }
 
 function preHandlerCallback (err, request, reply) {
-  if (reply.res.finished === true) return
+  if (reply.sent || reply.res.finished === true) return
```

Comment:
Any particular reason you're not using `reply.sent === true` here? (all of the other ones do so)

Reply:
that's what this change does :)